example/src/app/sdk/sdk.page.html

```diff
@@ -0,0 +1,109 @@
+<ion-header [translucent]="true">
+  <ion-toolbar mode="md">
+    <ion-thumbnail slot="start">
+      <img src="../../assets/icon/situm-logo-whitebg.svg" />
+    </ion-thumbnail>
+    <ion-title slot="start"> Situm Capacitor Example </ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content [fullscreen]="true">
+  <ion-card>
+    <ion-card-header>
+      <ion-card-title mode="md">
+        <ion-icon name="locate" color="primary"></ion-icon>
+        Positioning
+      </ion-card-title>
+    </ion-card-header>
+
+    <ion-card-content>
+      <ion-row>
+        <ion-button
+          size="small"
+          (click)="startPositioning()"
+          [disabled]="positioning"
+        >
+          startPositioning()
+        </ion-button>
+        <ion-button
+          size="small"
+          (click)="stopPositioning()"
+          [disabled]="!positioning"
+        >
+          stopPositioning()
+        </ion-button>
+      </ion-row>
+    </ion-card-content>
+  </ion-card>
+
+  <ion-card>
+    <ion-card-header>
+      <ion-card-title mode="md">
+        <ion-icon name="cloud-download" color="primary"></ion-icon>
+        Fetch Resources
+      </ion-card-title>
+    </ion-card-header>
+
+    <ion-card-content>
+      <ion-button size="small" (click)="fetchBuildingInfo()">
+        fetchBuildingInfo()
+      </ion-button>
+      <ion-button size="small" (click)="fetchPois()"> fetchPois() </ion-button>
+      <ion-button size="small" (click)="fetchPoiCategories()">
+        fetchPoiCategories()
+      </ion-button>
+      <ion-button size="small" (click)="fetchGeofences()">
+        fetchGeofences()
+      </ion-button>
+      <ion-button size="small" (click)="invalidateCache()">
+        invalidateCache()
+      </ion-button>
+    </ion-card-content>
+  </ion-card>
+
+  <ion-card>
+    <ion-card-header>
+      <ion-card-title mode="md">
+        <ion-icon name="map" color="primary"></ion-icon>
+        Cartography
+      </ion-card-title>
+    </ion-card-header>
+
+    <ion-card-content>
+      <ion-card>
+        <ion-item id="open-picker" *ngIf="!currentPoi"> Pick a POI </ion-item>
+        <ion-item id="open-picker" *ngIf="currentPoi">
+          {{ currentPoi.identifier }} - {{ currentPoi.poiName }}
+        </ion-item>
+      </ion-card>
+      <ion-picker
+        trigger="open-picker"
+        [columns]="pickerColumns"
+        [buttons]="pickerButtons"
+      ></ion-picker>
```

Comment:
Aquí aínda se me fai máis raro, porque estás pechando con </ion-picker>. Si a etiqueta non ten contido, podes simplemente pechala:
```
<ion-picker
    trigger="open-picker"
    [columns]="pickerColumns"
    [buttons]="pickerButtons"/>
```